toolbox/mount.c

```diff
@@ -328,8 +328,10 @@ int mount_main(int argc, char *argv[])
 	}
```

Comment:
All PRs should be submitted as Gerrit Changes. Please see the documentation at https://wiki.cyanogenmod.org/w/Doc:_using_gerrit